src/servers/daily-balance-notification.ts

```diff
@@ -1,8 +1,9 @@
-import { setupMongoConnection } from "@services/mongodb"
-import { User } from "@services/mongoose/schema"
+import { getRecentlyActiveAccounts } from "@app/accounts/active-accounts"
+import { getCurrentPrice } from "@app/prices"
+import { getBalanceForWallet } from "@app/wallets"
```

Comment:
this should be consumed from `@app` it is required if we want to have tracing from this

Reply:
how should this be called?

```
import Wallet from "@app"
Wallet.getBalanceForWallet()
```
?